efficientdet/utils.py

```diff
@@ -111,35 +111,47 @@ def get_ckpt_var_map(ckpt_path, ckpt_scope, var_scope, skip_mismatch=None):
   ckpt_var_name_to_shape = reader.get_variable_to_shape_map()
   ckpt_var_names = set(reader.get_variable_to_shape_map().keys())
 
+  if tf.distribute.get_replica_context():
+    replica_id = tf.keras.backend.get_value(
+      tf.distribute.get_replica_context().replica_id_in_sync_group)
+  else:
+    replica_id = 0
+
   for i, v in enumerate(model_vars):
-    if not v.op.name.startswith(var_scope):
+    var_op_name = v.op.name
+
+    if replica_id >= 1:
+      var_op_name = ''.join(var_op_name.rsplit(f'/replica_{replica_id}', 1))
```

Comment:
Good idea! Thanks @fsx950223

Reply:
For mirror strategy and replica_x, does x start from 0 or 1?

Reply:
replica_x starts at 1.
For the host, there is no replica_x in variable.op.name